agency/README.md

```diff
@@ -2,26 +2,30 @@
 
 <a href="/agency/"><img src="https://i.imgur.com/HzMWtaI.png" width="120" align="right" alt="MDS Agency Icon" border="0"></a>
 
-The Agency API endpoints are intended to be implemented by regulatory agencies and consumed by mobility providers. Providers query the Agency API when events (such as a trip start or vehicle status change) occur in their systems.
+The Agency API endpoints are intended to be implemented by regulatory agencies and consumed by mobility providers. Data is **pushed** to agencies by providers. Providers query the Agency API when events (such as a trip start or vehicle status change) occur in their systems.
 
 This specification contains a collection of RESTful APIs used to specify the digital relationship between *mobility as a service* providers and the agencies that regulate them.
 
 ## Table of Contents
 
 * [General Information](#general-information)
   * [Versioning](#versioning)
+  * [Modes](#modes)
   * [Responses and Error Messages](#responses-and-error-messages)
   * [Authorization](#authorization)
   * [GBFS](#gbfs)
 * [Vehicles](#vehicles)
   * [Vehicle - Register](#vehicle---register)
   * [Vehicle - Update](#vehicle---update)
-  * [Vehicle - Events](#vehicle---event)
-  * [Vehicle - Telemetry](#vehicle---telemetry)
+  * [Vehicles - Events](#vehicles---events)
```

Comment:
@marie-x Can we rename the Agency vehicle events endpoint from /vehicles/events to just /events? And /vehicles/telemetry to just /telemetry? Would provide symmetry with Provider endpoint names. Adding vehicles seems redundant since it is implied and is described in the docs, just like in Provider, and we don't for example say /vehicle/trips. Is there any reason to keep it for events and telemetry now?